README.md

```diff
@@ -28,7 +28,7 @@ C++ library by Rick Blommers.
 Using `conda`:
 
 ```
-conda install qtawesome
+conda install -c conda-forge qtawesome
```

Comment:
Mmmh, I don't think this change is necessary because it'd lead people to have package resolution issues if they run it in their base envs. What do you think?

Reply:
```suggestion
conda install qtawesome
```